package.json

```diff
@@ -11,6 +11,7 @@
     "sync": "node ./scripts/sync.js",
     "gclient": "node ./scripts/gclient.js",
     "build": "node ./scripts/commands.js build",
+    "check_deps": "node ./scripts/commands.js build --check_deps",
```

Comment:
why is this run as an option for build? Seems like it would be better to add a generic `gn` command wrapper like `gclient` or at the very least not connect this to the build.

Reply:
and actually this will completely screw up any previous build that has different params and cause large rebuilds

Reply:
also on CI this will trigger another build for Component build type